backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/IInvoiceHeader.java

```diff
@@ -63,8 +62,17 @@
 
 	int getM_InOut_ID();
 
+	@Nullable
 	I_C_DocType getC_DocTypeInvoice();
 
+	@NonNull
+	Optional<DocTypeInvoicingPoolId> getAggregatedDocTypeInvoicingPoolId();
+		
+	@NonNull
+	Optional<I_C_DocType> getSingleDocType();
```

Comment:
Q: can't we "merge" this 3 doc type fields
and use `DocTypeInvoicingPoolId` only?

in case a pool is not available, we can create one where negative/positive amt doc types are the one from single doc type

wdyt?

Reply:
merged